src/RestServer/Controllers/v1/LedgerController.cs

```diff
@@ -0,0 +1,389 @@
+        /// <returns>An array of witness object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/witnesses", Name = "GetTransactionWitnesses")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Witness[]))]
+        public IActionResult GetTransactionWitnesses(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Witnesses);
+        }
+
+        /// <summary>
+        /// Gets the signers of a transaction.
+        /// </summary>
+        /// <param name="hash" example="0xad83d993ca2d9783ca86a000b39920c20508c8ccae7b7db11806646a4832bc50">Hash256</param>
+        /// <returns>An array of Signer object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/signers", Name = "GetTransactionSigners")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Signer[]))]
+        public IActionResult GetTransactionSigners(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Signers);
+        }
+
+        /// <summary>
+        /// Gets the transaction attributes of a transaction.
+        /// </summary>
+        /// <param name="hash" example="0xad83d993ca2d9783ca86a000b39920c20508c8ccae7b7db11806646a4832bc50">Hash256</param>
+        /// <returns>An array of the transaction attributes object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/attributes", Name = "GetTransactionAttributes")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(TransactionAttribute[]))]
+        public IActionResult GetTransactionAttributes(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Attributes);
```

Comment:
Need show more critical details such as `txhash` for conflict attributes, `id`, `response` for oracle attributes.
![1702280467312](https://github.com/neo-project/neo-modules/assets/38493437/b8343a34-d366-473c-8656-4ff3f1ed3a94)